contracts/lib/MessageBus.sol

```diff
@@ -683,31 +683,20 @@ library MessageBus {
         pure
         returns(bytes32 storagePath_)
     {
+        if(_offset > 0){
+            _structPosition = _structPosition + _offset;
```

Comment:
Do we really need `_structPosition` and `_offset` as separate parameters if we don't use them anywhere except to add them?

Reply:
Conceptually these are two different values. 
Struct position is the index of struct in the storage and offset is the index inside the struct. 

@deepesh-kn  what do you think?